### PR TITLE
Add ssh_tunnel method

### PR DESCRIPTION
**Pull request by thesquelched**
Add method to create SSH tunnels, e.g. `lava clusters ssh_tunnel <id> <local port> <remote port> --component HiveServer2`

### CODE REVIEW

File: lavaclient/api/clusters.py [hunk 1]
```diff
@@ -720,3 +725,76 @@ def ssh_execute(self, cluster_id, node_name, command, ssh_command=None,
...
+        if node_name and component:
+            raise error.InvalidError(
+                'node_name and component are mutually exclusive')
+        elif not (node_name or component):
+            raise error.InvalidError(
+                'One of node_name or component is required')
+
+        cluster, nodes = self._cluster_nodes(cluster_id, wait=wait)
+
+        if component:
```
Comment (jimbobhickville) [inline lavaclient/api/clusters.py]: If I'm a customer, I would think specifying a component would tunnel me to the right port as well.  I don't think we expose that information as of yet, but something to keep in mind for the future.

**Comment by jimbobhickville** [comment]
+1

File: lavaclient/api/clusters.py (hunk 1 above)
Comment (thesquelched) [inline lavaclient/api/clusters.py]: I'll keep it in mind

File: lavaclient/api/clusters.py [hunk 2]
```diff
@@ -720,3 +725,76 @@ def ssh_execute(self, cluster_id, node_name, command, ssh_command=None,
...
+                                'Mutually exclusive with --node-name'),
+        ssh_command=argument(help="SSH command (default: 'ssh')"),
+        wait=argument(action='store_true',
+                      help="Wait for cluster to become active (if it isn't "
+                           "already)"),
+    )
+    def ssh_tunnel(self, cluster_id, local_port, remote_port, node_name=None,
+                   component=None, ssh_command=None, wait=False):
+        """
+        Create a SSH tunnel on from the local host to a particular port on a
```
Comment (rnirmal) [inline lavaclient/api/clusters.py]: Minor: Don't think you meant to say `tunnel on from`, just `SSH tunnel from`.

File: lavaclient/api/clusters.py [hunk 3]
```diff
@@ -720,3 +725,76 @@ def ssh_execute(self, cluster_id, node_name, command, ssh_command=None,
...
+
+        cluster, nodes = self._cluster_nodes(cluster_id, wait=wait)
+
+        if component:
+            ssh_node = self._get_component_node(nodes, component)
+        else:
+            ssh_node = self._get_named_node(nodes, node_name=node_name)
+
+        printer = self._cli_printer(LOG)
+        printer('Starting SSH tunnel from {0}:{1}:{2} via node {3} '
```
Comment (rnirmal) [inline lavaclient/api/clusters.py]: Would be better to say `Starting SSH tunnel from localhost:{0} to {1}:{2}`

**Comment by rnirmal** [comment]
+1, some minor comment changes and it can be merged.

**Pull request merged**